Habitica/src/com/magicmicky/habitrpgwrapper/lib/models/tasks/Task.java

```diff
@@ -112,8 +117,10 @@ public void setId(String id) {
     /**
      * @return the notes
      */
-    public String getNotes() {
-        return notes;
+    public CharSequence getNotes() {
+        CharSequence processed = Html.fromHtml(processor.markdownToHtml(EmojiParser.parseEmojis(notes)));
```

Comment:
That's what I wanted at first, but I didn't manage to get it working so I implemented it into the `Task`. I'm rewriting the Emoji-parser, so it will take some time to finish this. Should I close the PR and open a new one when I've finished the work or just leave this one open and contact you when I've finished?